src/osvr/Util/LogRegistry.cpp

```diff
@@ -187,7 +205,7 @@ namespace util {
             // File sink - rotates daily
             std::string logDir;
             try {
-                size_t q_size = 1048576; // queue size must be power of 2
+                size_t q_size = 65536; // queue size must be power of 2
```

Comment:
what I couldn't figure out, was what the units here are - characters? lines? We shouldn't be logging that much data that we need a massive queue.